### PR TITLE
Update .travis.yml to run omniscidb server

### PR DESCRIPTION
**Pull request by guilhermeleobas**
trying to setup omniscidb server on travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,16 +18,23 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update --quiet --yes conda
   - conda info -a
+  - conda config --add channels conda-forge
   - conda env create --quiet --name rbc-ci --file .conda/environment.yml
```
Comment (pearu) [inline .travis.yml:22]: rbc and omniscidb need likely different environments. So, I suggest to try the following approach:
```
conda env create --quiet --name rbc-ci --file .conda/environment.yml
conda env create --quiet --name omnisci-ci omniscidb
conda activate rbc-ci
...
```
and remove the `conda install omniscidb -c conda-forge` below.

Then:
```
before_script:
  - conda activate omnisci-ci
  - mkdir data && omnisci_initdb data -f
  - omnisci_server --enable-runtime-udf --enable-table-functions &
  - conda deactivate omnisci-ci
```

**Comment by guilhermeleobas** [comment]
PR is ready for review. There is an error raised by calcite which I cannot reproduce on omnisci 5.3 or omnisci master:
```
rbc/tests/test_omnisci.py::test_loadtime_udf 2020-07-03 23:56:08 ERROR CalciteServerHandler:process:177 - Validation: org.apache.calcite.runtime.CalciteContextException: From line 1, column 14 to line 1, column 36: No match found for function signature udf_diff2(<NUMERIC>, <NUMERIC>)
```

Also, should we stop pytest on first failure?

**Comment by pearu** [comment]
> Also, should we stop pytest on first failure?

Yes, especially with omnisci tests when the first test crashes the server.

File: .travis.yml [hunk 2]
```diff
@@ -27,7 +29,15 @@ install:
 
 before_script:
   - python setup.py develop
+  - conda activate omnisci-ci
+  - mkdir data && omnisci_initdb data -f 
+  - omnisci_server --enable-runtime-udf --enable-table-functions &
```
Comment (pearu) [inline .travis.yml:34]: Suggestion:
```
omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 > omniscidb-output.txt &
```

File: .travis.yml [hunk 3]
```diff
@@ -27,7 +29,15 @@ install:
...
+  - conda activate rbc-ci
 
 script:
   - flake8 .
-  - pytest -v -r s rbc/
\ No newline at end of file
+  - pytest -v -r s rbc/
+
+after_script:
+  - pkill -f omnisci_server
```
Comment (pearu) [inline .travis.yml:43]: Continued suggestion:
```
cat omniscidb-output.txt
```

File: .travis.yml (hunk 2 above)
Comment (guilhermeleobas) [reply .travis.yml]: Do we actually need to pipe the output to a file? It seems that travis doesn't support piping inside an `if` clause:
`if [[ some_condition ]]; omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 > omniscidb-output.txt &`

Comment (guilhermeleobas) [reply .travis.yml]: see: https://travis-ci.org/github/xnd-project/rbc/builds/705525843

Comment (pearu) [reply .travis.yml]: The build log shows command
```
if [ "$OMNISCIDB_FROM" = "conda" ]; then omnisci_server --enable-runtime-udf --enable-table-functions &> omniscidb-output.txt & ; fi
```
but it should be
```
if [ "$OMNISCIDB_FROM" = "conda" ]; then omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 omniscidb-output.txt & ; fi
```

Comment (pearu) [reply .travis.yml]: I think you just had a typo: `&>` ought to be `2>&1`

Comment (guilhermeleobas) [reply .travis.yml]: I have tried that as well. See https://travis-ci.org/github/xnd-project/rbc/jobs/705503755

Comment (pearu) [reply .travis.yml]: Create a `start_omniscidb_bg.sh` script that contains all relevant commands needed to setup and start the omniscidb server.

**Comment by guilhermeleobas** [comment]
I've added conda and docker omnisci versions to travis CI.

**Pull request merged**